app/src/main/java/me/morglenrazor/imdbcollector/presentaiton/base/BaseView.kt

```diff
@@ -1,9 +1,12 @@
 package me.morglenrazor.imdbcollector.presentaiton.base
 
+import android.view.View
 import moxy.MvpView
+import moxy.viewstate.strategy.StateStrategyType
 import moxy.viewstate.strategy.alias.AddToEndSingle
+import java.lang.Error
 
 interface BaseView : MvpView {
     @AddToEndSingle
```

Comment:
If you show an error, and then rotate your phone - error shows again. U need another annotation

Reply:
Alright, but i just forgot delete this import